### DISCUSSION TITLE
remote: Repository not found.

### DISCUSSION THREAD

**Discussion by thomasbinny** [community/community#55193] [New to GitHub]
Error: remote: Repository not found.

The repo is private, im able to clone using Github Desktop, but not using terminal.

Please suggest.

**Comment by hrishikesh-kocharekar-dev** [comment]


**Reply by airtower-luna** [reply]
The reply looks not only like ChatGPT nonsense, the suggestion to include credentials in the URL (by the way, the suggested ones wouldn't work) is a security risk. :warning:

**Comment by airtower-luna** [comment]
> The repo is private, im able to clone using Github Desktop, but not using terminal.
> 
> Please suggest.

I suggest you show the *full* terminal output. :wink: You can redact the repository name and other private details, but without the exact error message it's hard to guess what may be wrong.
